documentation/specs/DataGrid.md

```diff
@@ -96,6 +105,14 @@ type DataGridProps<C extends Column> = AriaLabelingProps & {
 
   /** The current sorted column and direction. */
   sortDescriptor?: SortDescriptor;
+
+  /**
+   * Define a custom grid-template-columns.
+   * Defaults to taking up an even amount of space.
+   *
+   * {@link https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns}
```

Comment:
This is rendering in Storybook with `}` as part of the link which is a 404
![image](https://github.com/EasyPost/easy-ui/assets/1400051/f8fa3ab0-57d5-456e-8962-8bd5ceeae1ee)

Reply:
i fixed the 404 by giving the URL a label, which is still valid JSDoc. unfortunate that Storybook doesn't parse it correctly